### PR TITLE
 Finish the project structure

### PR DESCRIPTION
**Pull request by majdalkilany**
- create the project structure 
- add some style 
- work in git-flow

### CODE REVIEW

**Review by AbelG101 — COMMENTED** [review]
# Hi @majdalkilany  👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- You have submitted a project, that doesn't have the proper linters configured specifically the Lighthouse check is missing so please make sure to properly configure it.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 1]
```diff
@@ -1 +1,119 @@
...
+
+- leaderBoard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://majdalkilany.github.io/To-Do-List/dist/)
```
Comment (divinecharlotte) [inline README.md]: - [ ] kindly consider adding the right path to your live demo of this project or consider removing this section if you decide to not deploye this project

File: README.md [hunk 2]
```diff
@@ -1 +1,119 @@
...
+
+- [ ] Add a navigation System
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/SiveMdluli/Leaderboard/issues).
```
Comment (divinecharlotte) [inline README.md]: - [ ]  kindly consider adding the right path to the issue page of this repo to make your readme more interactive and professional

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @majdalkilany   , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- Descriptive PR👍 
- Descriptive readme 👍 

 




 

## Required Changes ♻️

- [ ] kindly consider following the below structure to make sure that the dist folder is present as it contains the minimized version of the source code and actually it contains the code which is used on production web applications.
```
app-root-directory
  |- package.json
  |- webpack.config.js
  |- /dist
    |- bundle.js or be main.js
    |- index.html
  |- /src
   |- style.css
    |- index.js
  |- /node_modules

```

_Check the comments under the review._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by majdalkilany** [comment]
Add the dist directory 

**Note the Live URL will not work before merging the branches**

**Review by omarsalem7 — APPROVED** [review]
Hi @majdalkilany 😊, 
# STATUS: APPROVED! 🥇 💯 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**